modules/@apostrophecms/doc-type/index.js

```diff
@@ -593,7 +593,7 @@ module.exports = {
       async findOneForEditing(req, criteria, builders) {
         const query = await self.findForEditing(req, criteria, builders);
         const doc = query.toObject();
-        if (options.annotate) {
+        if (self.options.annotate) {
```

Comment:
What does this option do?

Reply:
It never worked and was never documented, plus we have the `attachments` query builder now as well as REST APIs that automatically take care of annotating attachments, so I'm removing it. Good catch.